src/services/profile.ts

```
@@ -0,0 +1,160 @@
+import { gql, ApolloClient } from '@apollo/client';
+import { UserProfileResponseDto, ChangeProfileRequest, FollowUserInfoResponseDto, FollowUserInfo, FollowerUserInfo, PasswordChangeInput } from '@/types/profile';
```

ESLint

[error] 'ChangeProfileRequest' is defined but never used.

(@typescript-eslint/no-unused-vars)